git-ftp

```diff
@@ -1285,8 +1292,16 @@ set_current_branch() {
 set_stash() {
 	local stash_config="$(get_config auto-stash)"
 	[ -n "$stash_config" ] && AUTO_STASH="$(boolean $stash_config)"
-	[ $AUTO_STASH -eq 1 ] && TMP_STASH_TOKEN="$(date +%s)"
-	[ $AUTO_STASH -eq 1 ] && git stash push --include-untracked -q -m "Stashed by Git-ftp. Token: $TMP_STASH_TOKEN"
+
+	if [ $AUTO_STASH -eq 1 ]; then
+		TMP_STASH_TOKEN="$(date +%s)"
+		local stash_message="Stashed by Git-ftp. Token: $TMP_STASH_TOKEN"
+		if [ "${GIT_VERSION[0]}" -le 2 ] && [ "${GIT_VERSION[1]}" -lt 13 ]; then
```

Comment:
Hm, we are relying on Git not to publishing version 1.13. I guess that's a save bet. ;-)

Reply:
Yeah, the version check in `check_git_version` freaked me out. I thought it would allow 1.8 first, but it doesn’t. So I decided to go a similar way here to keep it simple.